aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml

```diff
@@ -95,7 +95,6 @@ Resources:
                   STORAGE_PROVIDER=${BucketProvider}
                   BUCKET_NAME=${BucketName}
                   FOLDER_NAME=${BucketFolderName}
-                  BPF_FILTER=udp and port 4789
```

Comment:
make sure the sniffer does not have "default bpf filter"

Reply:
it doesn't have